### DISCUSSION TITLE
Sample exports file not found

### DISCUSSION THREAD

**Discussion by boomer4d** [microsoft/finops-toolkit#726] [Q&A] [answered]
I was trying to download the sample exports file from the Open Data page but am getting a file not found error.

https://microsoft.github.io/finops-toolkit/data#%EF%B8%8F-sample-exports

Because the cost management exports aren't supported in CSP for Azure Gov subscriptions I'm trying to see what that export looks like to see if it is comparable to the information I get from the Get-PartnerCustomerSubscriptionUtilization cmdlet which is supported and does work.

Thanks!

**Comment by flanakin** [comment, marked as answer]
I just saw this as well. The file was renamed and the link wasn't updated. I've already fixed it for the next release. Sorry for the inconvenience! You can download it here: https://github.com/microsoft/finops-toolkit/releases/download/v0.3/sample-data.zip

The data will be comparable, but different than what you get from Partner Center.
